### PR TITLE
This is for adding swagger for OperationsManagement RP.

### PR DESCRIPTION
**Pull request by dashimi16**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: arm-operationsmanagement/compositeOperationsManagement.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "info": {
+    "title": "Azure Log Analytics - Operations Managements",
+    "description": "Azure Log Analytics API reference for Solution."
+  },
+  "documents": [
+    "./2015-11-01-preview/swagger/OperationsManagement.json",
```
Comment (vishrutshah) [inline arm-operationsmanagement/compositeOperationsManagement.json]: Why are you adding two api-version of same RP into composite swagger?

File: arm-operationsmanagement/2017-03-03-preview/Swagger/OperationsManagement.json [hunk 2]
```diff
@@ -0,0 +1,348 @@
+{
```
Comment (vishrutshah) [inline arm-operationsmanagement/2017-03-03-preview/Swagger/OperationsManagement.json]: Could you please follow the guidelines outlined https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md#filenames-and-folder-structure .

Please make sure you create 3 commits. 
One with just api-version `2015-11-01-preview` 
Second with copy of `2015-11-01-preview` api-version into folder name `2017-03-03-preview`
Third with updating the only changes into the new api-version. We should only review the diffs between api-versions. 
Fourth with the adding [x-ms-examples](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/x-ms-examples.md#why-x-ms-examples). 

 That's greatly help us in reviewing the changes. Thanks.

**Review by vishrutshah — CHANGES_REQUESTED** [review]

**Comment by vishrutshah** [comment]
Some general questions:

- [ ] Was this RP reviewed by One API & ARM Review team?
- [ ] Are both the version of api live i.e. deployed in production?

**Comment by vishrutshah** [comment]
@dashimi16 Make sure to sign the cla if you haven't already.

File: arm-operationsmanagement/2017-03-03-preview/Swagger/OperationsManagement.json (hunk 2 above)
Comment (dashimi16) [inline arm-operationsmanagement/2017-03-03-preview/Swagger/OperationsManagement.json]: I will cancel this pull request and then create one with at a time.

**Comment by dashimi16** [comment]
Closing this pull request since got feedback for smaller commit. Will make that change and publish a new pull request. Thanks for your time in reviewing the pull request.

**Pull request closed without merging**